jme3-lwjgl3/src/main/java/com/jme3/system/lwjgl/LwjglWindow.java

```diff
@@ -291,6 +289,8 @@ public void invoke(int error, long description) {
             requestWidth = videoMode.width();
             requestHeight = videoMode.height();
         }
+        oldFramebufferHeight = requestHeight;
```

Comment:
By assuming that the scaling factor of 1 is correct we avoid extra resize at the start of application, also fixes TestAwtPanels. If you really don't have scaling... AwtContext wasn't build scaling in mind. It throws an exception when framebuffer resize is triggered, and that is how our scaling currently works.

Reply:
Please see: https://github.com/jMonkeyEngine/jmonkeyengine/pull/1950/files#r1103859250

Reply:
> TestAwtPanels throws an exception

AwtPanelsContext is modified to log an error instead of throwing an exception which should make TestAwtPanels work. 

Please see https://github.com/jMonkeyEngine/jmonkeyengine/pull/1949